modules/concept-docs/pages/data-durability-acid-transactions.adoc

```diff
@@ -159,6 +159,52 @@ This durability is enforced by the database,
 // https://docs.couchbase.com/java-sdk/current/concept-docs/durability-replication-failure-considerations.html#older-server-versions
 // Change link once 3.3 is archived?
 
+
+
+
+== Reading from Replicas
+
+By default, read requests from the SDK are directed to the node with the active vbucket.
+This bucket will usually be updated before any replicas (see <#durability,Durability> above),
+and is best for consistency.
+
+Some use cases benefit -- for consistency or availability -- from accessing both the active copy of the document, and/or one or all replicas.
+This can be from the {anyreplica-api}[data service],
+and xref:howtos:subdocument-operations.adoc#reading-sub-documents-from-replicas[SubDoc Read from Replica].
+The method can either be one to return either the first result returned -- active or replica --
+which is useful if a node is timing out;
+or to return _all_ of the results, for client code to handle any inconsistency, or to build a consensus answer.
+
+=== Local Availability Zone Reads
+
+IMPORTANT: Zone Aware Replica Reads are only accessible with the {name-sdk} working with Couchbase Server 7.6.2 or newer (Capella or self-managed).
+
+For Capella users with high data volumes, egress charges for reads from other Availability Zones (AZ) in AWS can be a significant cost.
+The {name-sdk}, when making read replica requests, can be zone aware --
+and set to always read from a copy of the document in the local zone.
+This is done by putting cluster nodes in the same AZ into the same 
+xref:server:learn:clusters-and-availability/groups.adoc#server-groups-and-vbuckets[Server Group], too.
+
+This may mean the application has to be tolerant of slight inconsistencies,
+until the local replica catches up.
+Alternatively, it may demand a stronger level of durability, to ensure that all copies of a document are consistent before they are accessible --
+provided that this is `persistToMajority` with xref:server:learn:data/durability.adoc#majority[no more than one replica].
+
+In any case, Couchbase does not recommend this feature where read consistency is critical.
```

Comment:
We may want to word this better because it may cause more concern than we intend.